package.json

```diff
@@ -305,6 +305,8 @@
     "csv-parse": "^4.14.1",
     "csv-stringify": "^5.6.2",
     "cucumber-html-reporter": "^5.2.0",
+    "deepmerge": "^4.3.1",
+    "dot-wild": "^3.0.1",
```

Comment:
```dot-wild``` is a little worrying to me, there are security related issues on the repo and it hasn't been updated in 6 years

Reply:
A few more files down, I see it would be difficult to replace as it seems to be deeply intertwined with the approach we took here

Reply:
https://security.snyk.io/package/npm/dot-wild I don't see any vulnerabilities. What issues do you see?

Reply:
It had not been updated in 6 years, agreed. But it works great.

Reply:
@thewatermethod What concerns do you have other then it has not been touched in 6 years?

Reply:
I was referring to this: https://github.com/wadackel/dot-wild/issues/18